Packs/ServiceNow/Integrations/ServiceNow_CMDB/ServiceNow_CMDB.py

```diff
@@ -547,11 +551,31 @@ def main() -> None:
     client_id = client_secret = ""
     credentials = params.get("credentials", {})
     use_oauth = params.get("use_oauth", False)
+    use_jwt = params.get("use_jwt", False)
+    jwt_params = {}
+
+    # use jwt only with OAuth
+    if use_jwt and use_oauth:
```

Comment:
I didn't understand the logic here we check that they are not both selected but if just use_jwt we set the use_oauth true?